infra/core/monitor/applicationInsights.bicep

```diff
@@ -1,25 +1,30 @@
-param environmentName string
-param suffix string = 'linter'
+metadata description = 'Creates an Application Insights instance based on an existing Log Analytics workspace.'
```

Comment:
this file name needs to be applicationinsights.bicep, not applicationInsights.bicep